src/test/demo-sample.spec.ts

```diff
@@ -9,6 +9,6 @@ test("demo sample document is rendered correctly when dragged", async (t) => {
   await t
     .expect(Selector("[data-testid='banner-title']").withText("Want to try creating a verifiable document?").exists)
     .ok();
-  await validateIssuerTexts(["EXAMPLE.OPENATTESTATION.COM"]);
+  await validateIssuerTexts(["DEMO.TRADETRUST.IO"]);
```

Comment:
Oh... Is this testing the demo doc on the frontend? 'Cos I see the fixture docs here are issued under `demo-tradetrust.openattestation.com`.

Reply:
yes u r right its the demo doc. the demo doc i nv re-issue, keeping it as it was